### DISCUSSION TITLE
UI垮掉了, 视频区域也特别小

### DISCUSSION THREAD

**Discussion by ANUBIS-V** [aidenlx/media-extended#323] [Q&A] [answered]
![image](https://github.com/PKM-er/media-extended/assets/963998/464c251a-d4f5-4df4-ac9f-e24344e4de3f)

**Comment by aidenlx** [comment]
什么途径安装的？手动的话确保styles.css有正确安装

**Reply by ANUBIS-V** [reply]
从市场安装的

**Reply by ANUBIS-V** [reply, marked as answer]
我去看了插件文件夹, 没有你说的css, 卸载重装好了
